src/components/ConfirmPopup.js

```diff
@@ -0,0 +1,30 @@
+import Popup from "./Popup";
+
+export default class ConfirmPopup extends Popup {
+  constructor(modalSelector) {
+    super(modalSelector);
+  }
+
+  setEventListeners() {
+    super.setEventListeners();
+    this._popup
+      .querySelector(".confirmation__button")
+      .addEventListener("click", () => {
+        if (this._submitFunction) {
+          this._submitFunction();
+        }
+      });
+  }
+
+  setSubmitFunction(submitFunction) {
+    this._submitFunction = submitFunction;
+  }
+
+  open() {
+    super.open();
+  }
+
+  close() {
+    super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.